mapfishapp/src/deb/resources/etc/georchestra/mapfishapp/print/config.yaml

```diff
@@ -0,0 +1,1026 @@
+---
+# Reference URL:
+# http://www.mapfish.org/doc/print/configuration.html
+
+#===========================================================================
+# allowed DPIs
+#===========================================================================
+dpis:
+  - 254
+  - 127
+  - 91
+
+#===========================================================================
+# the allowed scales
+#===========================================================================
+scales:
+  - 267
+  - 533
+  - 1066
+  - 2133
+  - 4265
+  - 8531
+  - 17062
+  - 34124
+  - 68247
+  - 136495
+  - 272989
+  - 545979
+  - 1091958
+  - 2183915
+  - 4367830
+  - 8735660
+  - 17471321
+  - 34942642
+  - 69885283
+  - 139770566
+  - 279541132
+  - 559082264
+
+# see https://github.com/georchestra/georchestra/issues/362
+brokenUrlPlaceholder: http://upload.wikimedia.org/wikipedia/commons/c/ce/Transparent.gif

```

Comment:
Was replaced by https://github.com/georchestra/template/blob/15.06/mapfishapp/WEB-INF/print/config.yaml#L40

I hope we won't forget other things like that ...